oar/core/notification_mgr.py

```diff
@@ -443,6 +459,52 @@ def get_slack_message_for_dropped_and_must_verify_bugs(
 
         return message
 
+    def get_slack_message_for_docs_and_prodsec_approval(
+        self, doc_appr, prodsec_appr
+    ):
+        """
+        manipulate slacke message for docs and prodsec approval
+
+        Args:
+            doc_appr ([]): list of no doc approved advisories
+            prodsec_appr ([]): list of no product security approved advisories
+
+        Returns:
+            str: slack message
+        """
+        gid = self.sc.get_group_id_by_name(
+            self.cs.get_slack_user_group_from_contact(
+                "approver", "doc_id"
+            )
+        )
+        userid = []
+        email_contact = self.cs.get_prodsec_email().split(",")
+        if len(email_contact) > 1:
+            for email in email_contact:
+                userid.append(self.sc.get_user_id_by_email(email))
+        else:
+            userid = email_contact
+        userid = " ".join(userid)
+        message = ""
+        if len(doc_appr)&len(prodsec_appr):
```

Comment:
```suggestion
        if len(doc_appr) or len(prodsec_appr):
```

Reply:
to make this logic much simple, we can manipulate doc_aporoval and prodsec approval message separately. i.e. 

```
message = ""
if len(doc_appr):
	# get message for doc approval

if len(prodsec_appr):
	# append message for prodsec approval
```